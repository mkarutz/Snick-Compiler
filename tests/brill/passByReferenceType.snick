# Types of actual parameters must be the same as formal parameters when passing
# by reference

proc main ()
    int x;

    foo(x);
end

proc foo (ref float x)
    write x;
end
