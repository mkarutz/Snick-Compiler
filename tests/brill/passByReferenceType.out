Failure: Type mismatch passing by reference.
Compilation Failed
